redaxo/src/addons/phpmailer/boot.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * PHPMailer Addon.
+ *
+ * @author markus[dot]staab[at]redaxo[dot]de Markus Staab
+ *
+ * @package redaxo\phpmailer
+ */
+
+if (!rex::isBackend() && $this->getConfig('errormailer') == true) {
+    
+    rex_extension::register('RESPONSE_SHUTDOWN', function(rex_extension_point $ep)
+    {
+        $logFile  = rex_path::coreData('system.log');
+        $fileTime = filemtime($logFile);
+        $sendTime = $this->getConfig('last_log_file_send_time', 0);
+        $timediff = $fileTime - $sendTime;
+        if (filesize($logFile) > 0 && $timediff > 9 && $file = new rex_log_file($logFile)) {
```

Comment:
Braucht es den filesize check oder kann man sich den in jedem request sparen?

Reply:
ich sah es als einfachste Lösung zu prüfen ob die Datei gefüllt ist.

Reply:
Eine idee, wie man es anders machen könnte? filesize geht sicher schneller als fopen

Reply:
Nur die reihenfolge der Bedingungen ändern reicht aus.